fw.json

```diff
@@ -7,7 +7,7 @@
         ["Flysky PL18", "pl18-"],
         ["Flysky PL18EV", "pl18ev-"],
         ["FrSky Horus X10", "x10-"],
-        ["FrSky Horus X10 Access", "x10-access-"],
+        ["FrSky Horus X10 Access", "x10express-"],
         ["FrSky Horus X12s", "x12s-"],
         ["FrSky QX7", "x7-"],
         ["FrSky QX7 Access", "x7-access-"],
```

Comment:
Is there any point dropping the hyphen here also (and everywhere else used also)?

Reply:
Absolutely